src/serlo-editor/math/editor.tsx

```diff
@@ -119,13 +119,21 @@ export function MathEditor(props: MathEditorProps) {
                 hover:bg-editor-primary-200 focus:bg-editor-primary-200 focus:outline-none
                 `}
               value={isVisualMode ? 'visual' : 'latex'}
+              data-qa="editor-plugin-toolbar-math-visual-latex-switch"
```

Comment:
Maybe it can be a bit shorter, like:
```suggestion
              data-qa="plugin-toolbar-math-type-switch"
```

.

\+ I'd use this one + the text of the options, to access the options. Three `data-qa` attributes for a select and its two static options seems like an overkill to me.

Both are just suggestions, of course.